dateutil/tz/win.py

```diff
@@ -12,7 +12,6 @@
     # ValueError is raised on non-Windows systems for some horrible reason.
     raise ImportError("Running tzwin on non-Windows system")
 
-from ._common import tzname_in_python2, _tzinfo
```

Comment:
This is no longer necessary because tzrangebase already includes these, good to go.